spec/components/email/sender_spec.rb

```diff
@@ -393,6 +400,7 @@
       it 'should create the right log' do
         email_sender.send
         expect(email_log.post_id).to eq(post.id)
+        expect(email_log.topic_id).to eq(topic.id)
         expect(email_log.topic.id).to eq(topic.id)
```

Comment:
The assertions here seems to be testing for the same thing.

Reply:
This will be removed in next PR, I just wanted to add a test in there to make sure that temporary topic method I added is fine

Reply:
:ok_hand: